nidaqmx_examples/Analog_Generation/Current/Cont_Gen_Cur_Wfm-Int_Clk.py

```diff
@@ -0,0 +1,267 @@
+"""
+Python DAQmx API program:
+	ContGen_IntClk
+
+Author: Andres Quesada
+Copyright 2021 National Instruments Corp.
+
+Example Category:
+    Analog Generation
+
+ Description:
+    This example demonstrates how to output a continuous number of
+    current samples to an Analog Output Channel using an internal
+    sample clock.
+
+ Instructions for Running:
```

Comment:
These instructions sound like there is an interactive UI or something. Can you change the language to make it clear that they need to modify the code?